### PR TITLE
Add Values & Composes examples

### PR DESCRIPTION
**Pull request by notrab**
Following on from our quick discussion on Twitter @jackfranklin, I've gone ahead and added an example using value and composes to this repo.

I'll publish to Medium an accompanying overview of the additions if you do wish to merge.

### CODE REVIEW

**Comment by jackfranklin** [comment]
@notrab how do you fancy the blog post being on JS Playground? Credited to you, of course. I'd love to publish that if you were interested. Else no worries, go for Medium 👍

File: src/App.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import styles from './app.css';
+// import {app, btnPrimary, btnSecondary}styles from './app.css'; // alt import
```
Comment (jackfranklin) [inline src/App.js]: can we remove this?

**Comment by notrab** [comment]
@jackfranklin that would be awesome. I'll make some time over the next few days to get it done and sent over in a PR.

**Comment by jackfranklin** [comment]
Great!

**Comment by jackfranklin** [comment]
@notrab just to confirm, if you're still happy to write the article I'll merge this just before that blog post goes out :)

**Comment by notrab** [comment]
Sorry @jackfranklin - It's been a busy weekend. I cloned the javascriptplayground repo and started it. I'll finish it ASAP. 🙄

**Comment by jackfranklin** [comment]
@notrab no no don't apologise! No need for ASAP, do it whenever you can :) Just wanted to let you know that was why I hadn't merged this :)

**Comment by jackfranklin** [comment]
@notrab ping ;)

**Comment by notrab** [comment]
It's been so long I can't even remember using css modules within Webpack 👀

**Comment by jackfranklin** [comment]
@notrab OK - I think we'll close this then? :)

**Pull request closed without merging**